pages/1.11/release-notes/1.11.5/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.5
+title: Release Notes for 1.11.5
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.5
+---
+
+DC/OS 1.11.5 was released on September 12, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Issues Fixed in DC/OS 1.11.5
+
+## Customer Advisory
+The customer advisory information is available [here](https://support.mesosphere.com/s/article/Critical-Issue-DC-OS-Networking-MSPH-2018-0003).
+
+**Note:** The [support website](https://support.mesosphere.com/s/downloads) requires a [login credential](https://support.mesosphere.com/s/login/) to access the customer advisory information. 
+
+## GUI
+- COPS-3051/DCOS-39662 - Fix deployment state in Marathon-LB debug page.
+- COPS-3573/DCOS-39720 - Enable service endpoints on UCR.
+- DCOS-21049 - Add `dcos:secrets:list:default:/` secret permission to access DC/OS UI.
+- DCOS-38595 - Fix display error messages in framework configuration form.
+- DCOS-39877 - Remove flickering filter issue in tasks view.
+- DCOS-40523 - Enable empty environment variables reducer to support empty values. 
+- DCOS-40577 - Fix dashboard permissions and node tab with correct ACLs for users (non-super users and remote users).
+- DCOS_OSS-3801 - Add support for Docker parameters in Metronome jobs.
+
+## Marathon
+- COPS-3505/MARATHON-8326 - Add `wipe=true` support for pod's instances endpoint to automatically drain and decomission agent nodes.
+- MARATHON-7568 - Prevent ZK credentials leak in /v2/info.
+- MARATHON-8084 - Clean up Marathon API POST header. 
+- MARATHON-8368 - Fix offer rejection statistics error when maintenance mode is enabled. 
+
+## Mesos
+- COPS-3616/DCOS-39973/DCOS-40162 - Fix container launch failures that occurred due to Mesos-bridge running out of IPs.
+- COPS-3750/DCOS-41224 - Fix for UCR container cleanup `EBUSY` issue, which restarts EdgeLB tasks.
+- DCOS-38225 - Prevent task loss by fixing error handling failures of Mesos-IAM interaction. [enterprise type="inline" size="small" /]
+- DCOS-40410 - Bump Mesos to nightly [1.5.x eacabd7](https://github.com/mesosphere/mesos/blob/eacabd7/CHANGELOG).
+
+## Networking
+- COPS-3279/COPS-3576/DCOS-37703/DCOS-39721 - Fix erroneous values in service addresses stats. Bump dcos-net.
+- COPS-3472/DCOS-38932 - Enable access to Marathon app via overlap network. 
+- COPS-3520/DCOS-39999 - Fix DC/OS OSS build failure that occurred due to segmentation violation.
+- DCOS-38600 - Remove deadlock when SSL sockets are simultaneously sending/receiving data.
+- DCOS-39707 - Fix clustering issues with `etcd`.
+- DCOS_OSS-3697 - Fix connectivity issue between bridged and overlay networks.
```

Comment:
Changed "bridged" to "bridge" (to be consistent with the docs).

Reply:
Done.